### PR TITLE
Change

### PR DESCRIPTION
**Pull request by Foinowl**
Hi, please approve my changes. I think these edits will improve the quality of your app. By, thks

### CODE REVIEW

File: demo/src/main/java/com/example/Bank/model/TransferBalance.java [hunk 1]
```diff
@@ -13,8 +13,14 @@ public class TransferBalance {
     private BigDecimal amount;
 
     public Long getTo() {
+        return to;
     }
 
-    public BigDecimal getAmout() {
+    public Long getFrom() {
```
Comment (mipo256) [inline demo/src/main/java/com/example/Bank/model/TransferBalance.java]: You already have `lombok.Data`, what is the purpose of declaring getters explicitly?

File: demo/src/main/java/com/example/Bank/BankService.java [hunk 2]
```diff
@@ -10,11 +10,9 @@
...
-    private final BalanceController repository;
+    private final BalanceRepository repository;
 
-
-
-    public BigDecimal getBalance(Long accountId){
+    public BigDecimal getBalance(Long accountId) throws IllegalAccessException {
         BigDecimal balance = repository.getBalanceForId(accountId);
         if (balance == null){
             throw new IllegalAccessException();
```
Comment (mipo256) [inline demo/src/main/java/com/example/Bank/BankService.java]: This exception is JDK 9+ modules API specific, you should use a custom one

File: demo.iml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mipo256) [inline demo.iml]: To `.gitignore`, unrelated here

File: banks/src/main/java/org/example/BankService.java [hunk 4]
```diff
@@ -0,0 +1,4 @@
+package org.example;
+
+public class BankService {
```
Comment (mipo256) [inline banks/src/main/java/org/example/BankService.java]: You already declared this classes in `com.example.Bank` package (fix naming in accordance to [google code](https://google.github.io/styleguide/javaguide.html) style btw), what is the purpose of declaring exact same classes here?

File: banks/src/main/java/org/example/BalanceRepository.java [hunk 5]
```diff
@@ -0,0 +1,4 @@
+package org.example;
+
+public class BalanceRepository {
```
Comment (mipo256) [inline banks/src/main/java/org/example/BalanceRepository.java]: same as for `BankService`

File: banks/pom.xml [hunk 6]
```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.example</groupId>
```
Comment (mipo256) [inline banks/pom.xml]: You should decide - you what  `org.example` as root  package or `com.example`

File: banks/banks.iml [hunk 7]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module version="4">
+  <component name="CheckStyle-IDEA-Module">
```
Comment (mipo256) [inline banks/banks.iml]: Remove from git also, no need to keep it here

**Review by mipo256 — COMMENTED** [review]
@Foinowl consider either to move you changes into demo folder, or create maven multi-module project.

**Comment by mipo256** [comment]
please, @frontend12, also take a look

File: demo/src/main/java/com/example/demo/BankService.java [hunk 8]
```diff
@@ -12,10 +13,10 @@
 public class BankService {
     private final BalanceRepository repository;
 
-    public BigDecimal getBalance(Long accountId) throws IllegalAccessException {
+    public BigDecimal getBalance(Long accountId) {
         BigDecimal balance = repository.getBalanceForId(accountId);
         if (balance == null){
-            throw new IllegalAccessException();
+            throw new ExceptionErrorBalance("There is no money on the balance");
```
Comment (serezakorotaev) [inline demo/src/main/java/com/example/demo/BankService.java:19]: If you want u can add ExceptionHandler :)

**Pull request closed without merging**